web/index.html

```diff
@@ -71,6 +71,10 @@ <h2 class="ts-dark">Infrastructure</h2>
         <h2 class="ts-dark">/switchboard</h2>
         <p>Quickly look up SCAN barcodes in REDCap.</p>
       </a>
+      <a class="bg-purple" href="/husky-musher/lead-dawgs">
+        <h2 class="ts-dark">Lead Dawgs</h2>
+        <p>Streamline UW Reopening kiosk testing.</p>
```

Comment:
```suggestion
        <p>Streamline Husky Coronavirus Testing kiosks.</p>
```

Let's strike "UW Reopening" from public-facing stuff in favor of the official name "Husky Coronavirus Testing".

Reply:
Makes sense to me!